cameronRomo/overlook
# Run lint to clean up code

## What is the change?
* Code is now linted.

## What does it fix?
* Unused definitions have been removed  and proper indentations were added
 
## Is this a fix or a feature?
* Fix

## Where should the reviewer start?
* Check each file and/or pull-down and run `npm run lint`

## How should this be tested?
* As you are observing the code base, notice the efforts made to make it clean and readable

## What are the relevant tickets?
Closes #28
